itk-module.cmake

```diff
@@ -21,7 +21,16 @@ itk_module(VariationalRegistration
     ITKRegistrationCommon
     ITKMathematicalMorphology
     ITKBinaryMathematicalMorphology
-    ITKTestKernel #necessary to handle IO in src
+    ITKIOGDCM
+    ITKIOMeta
+    ITKIOJPEG
+    ITKIOPNG
+    ITKIOTIFF
+    ITKIOBMP
+    ITKIOVTK
+    ITKIONRRD
+    ITKIOGIPL
+    ITKIONIFTI
```

Comment:
May be these could be replaced by `ITKImageIO`? Was also applied in [LesionSizingToolkit](https://github.com/InsightSoftwareConsortium/LesionSizingToolkit/pull/26) 😉

Reply:
:+1:

Reply:
I tried that, I was still getting some errors. But those might have been unrelated errors.

Reply:
I tried it again, and again it did not work ☹️

Reply:
Well, may be we just need to put this aside for now, and may be at a latter time we might be able to discover the reason :pensive:. Thanks for trying Dženan.